modules/dkan/dkan_dataset/dkan_dataset.module

```diff
@@ -1233,11 +1233,32 @@ function dkan_dataset_entity_info_alter(&$entity_info) {
  */
 function dkan_dataset_preprocess_node(&$vars) {
   if ($vars['view_mode'] == 'search_result') {
+    if (!empty($vars['body'][0]['summary'])) {
+      $body_summary = $vars['body'][0]['summary'];
+    }
+    else {
```

Comment:
Got `Notice: Undefined offset: 0 in dkan_dataset_preprocess_node() (line 1240 of /src/dkan/modules/dkan/dkan_dataset/dkan_dataset.module).` - may need a isset check on the body[0] in general.

Reply:
Good catch. I'll make the code change and resubmit.